libnetdata/log/log.c

```diff
@@ -886,3 +902,22 @@ void log_access( const char *fmt, ... ) {
             netdata_mutex_unlock(&access_mutex);
     }
 }
+
+#ifdef NETDATA_INTERNAL_CHECKS
+void log_aclk_message_bin( const char *data, const size_t data_len, int tx, const char *mqtt_topic, const char *message_name) {
+    if (aclklog) {
+        static netdata_mutex_t aclklog_mutex = NETDATA_MUTEX_INITIALIZER;
+        netdata_mutex_lock(&aclklog_mutex);
+
+        char date[LOG_DATE_LENGTH];
+        log_date(date, LOG_DATE_LENGTH);
+        fprintf(aclklog, "A %s C %s, Msg:\"%s\"(%s), MQTT-topic:\"%s\":\n", tx ? "->" : "<-", date, message_name, tx ? "tx" : "rx", mqtt_topic);
```

Comment:
Perhaps it is better to keep the timestamp consistent with the rest of the logfiles e.g. `2022-07-04 17:08:58:` and maybe follow with INCOMING or OUTGOING instead of A -> C and A <-- C

Reply:
that is the timestamp format `2022-07-01 13:23:57`, changing the A/C thing

Reply:
> that is the timestamp format `2022-07-01 13:23:57`, changing the A/C thing

yes with the ':'